docs/recipes/LocalFirstDataWithPowerSync.md

```diff
@@ -0,0 +1,713 @@
+---
+title: PowerSync for Local-First Data Management
+description: Enhance your app with PowerSync for efficient data synchronization between your app's local database and backend
+tags:
+  - PowerSync
+  - React Native
+  - Backend
+  - State management
+  - Database
+  - Data Synchronization
+  - Offline Support
+last_update:
+  author: Trevor Coleman
+publish_date: 2024-03-22
+---
+
+# PowerSync for Local-First Data Management
+
+This guide helps you integrate PowerSync with Supabase in an Ignite app for efficient local-first data management.
+
+[PowerSync](https://www.powersync.com/) allows your app to work smoothly offline while keeping the data in sync with your backend database.
+
+### What is Powersync?
+
+PowerSync is a service which synchronizes local data with your Postgres SQL back end. It lets your app work with a local copy of the users' data and automatically syncs changes to and from your backend. 
+
+Because your application interacts with the data in a local instance of SQLite, it means you'll always have fast, responsive access -- no fetching, no spinners. It also means 
+your users will have a seamless, consistent experience even if they are offline.
+
+In the background, Powersync queues any changes and syncs the local data with the server whenever an internet connection becomes available. That means the data stays up to date across all of their devices.
+
+### Benefits of Using Powersync
+
+* **Handles Intermittent Network Connectivity**: Powersync allows your app to remain operational even in areas with
+  unreliable internet access. Users can continue their tasks without interruption, with automatic syncing when the
+  connection is restored.
+* **Enables Offline Operation**: With Powersync, your application can fully function offline, allowing users to access
+  and interact with their data regardless of their internet connection status.
+* **Eliminates Loading Delays**: Leveraging local data minimizes the need for loading indicators, offering a smoother,
+  faster experience for the user.
+* **Supports Multi-device Sync**: Powersync ensures data consistency across all of a user's devices, enabling seamless
+  access and transition between different platforms.
+  By integrating Powersync into your Ignite project, you provide a more reliable and user-friendly experience, ensuring
+  your application remains functional and responsive under various network conditions.
+
+## Prerequisites
+
+1. **An Ignite app using `Expo CNG` or `Bare` workflow**
+   - PowerSync requires native modules, so you cannot use Expo Go
```

Comment:
We can give a copy/pastable command to generate this real quickly

```
```bash
npx ignite-cli@latest new PizzaSyncApp --remove-demo --workflow=cng --yes
cd PizzaApp
```